assets/oauth_clients/components/EditClient.jsx

```diff
@@ -51,6 +51,7 @@ class EditClient extends React.Component {
                                         name='client secret'
                                         label={gettext('Client Secret')}
                                         value={this.props.client.secret_key}
+                                        iconName='icon--copy'
```

Comment:
imo we should rather have a property like `copyAction={true}`, it's not really about icons